src/notification/index.js

```diff
@@ -65,25 +65,30 @@ export const getAccountFromNotificationData = (
       urlMatches.push(i);
     }
   });
+
   if (urlMatches.length === 0) {
-    // No match.  Either we logged out of this account and didn't
-    // successfully tell the server to stop sending notifications (possibly
-    // just a race -- this notification was sent before the logout); or
-    // there's some confusion where the realm_uri we have is different from
-    // the one the server sends in notifications.
+    // No match.  Either we logged out of this account and didn't successfully
+    // tell the server to stop sending notifications (possibly just a race --
+    // this notification was sent before the logout); or there's some confusion
+    // where the realm_uri we have is different from the one the server sends in
+    // notifications.
     const knownUrls = identities.map(({ realm }) => realm);
-    logging.warn(
-      `notif realm_uri not found: ${realm_uri}, known values are ${JSON.stringify(knownUrls)}`,
-    );
+    logging.warn('notification realm_uri not found in accounts', {
+      notification_uri: realm_uri,
+      known_urls: knownUrls,
+    });
     return null;
   }
 
   if (urlMatches.length > 1) {
-    // The user has several accounts in the notification's realm.  We should
-    // be able to tell the right one using the notification's `user_id`...
-    // except we don't store user IDs in `accounts`, only emails.  Until we
-    // fix that, just ignore the information.
-    logging.warn(`notif realm_uri ambiguous: ${urlMatches.length} matches for ${realm_uri}`);
+    // The user has several accounts in the notification's realm.  We should be
+    // able to tell the right one using the notification's `user_id`... except
+    // we don't store user IDs in `accounts`, only emails.  Until we fix that,
+    // just ignore the information.
+    logging.warn('notification realm_uri ambiguous; multiple matches found', {
+      match_count: urlMatches.length,
+      realm_uri,
```

Comment:
nit: Let's use the same name for the "`realm_uri` from notification" value in both the "<1 matches" and ">1 matches" cases.